Contributors/AlphaBah/twoSums.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable indent */
+
+
+//Two Sum
+// Given an array of integers, return indices of the two numbers such that they add up to a specific target.
+
+// You may assume that each input would have exactly one solution, and you may not use the same element twice.
+
+//     Example
+// Given nums = [2, 7, 11, 15] and target = 9, return [0, 1] because nums[0] + nums[1] = 2 + 7 = 9.
+
+const twoSum = (nums, target) => {
+
+    // make sure that there are values:
+    if (!nums || !target && nums === undefined || target === undefined) {
+        return console.log("add values")
+    }
+
+    //remove repeated elements
+    const removeRepeatedElmnt = (nums) => {
+        return nums.filter((item, index) => nums.indexOf(item) === index)
+
+    }
+    const filteredElements = removeRepeatedElmnt(nums)
+
+
+    let indecise = []
+    for (let i = 0; i < filteredElements.length; i++) {
+        const nextValueOnTheRight = filteredElements[i + 1]
+        const addedItems = filteredElements[i] + nextValueOnTheRight
```

Comment:
What happens if two numbers add up to the target but aren’t next to each other?

e.g. `nums = [2, 15, 7, 11]` and `target = 9`, we’d expect the result to be [0, 2]. Would your function return the right value? If not, how could you fix it?